lib/attribute.js

```diff
@@ -163,6 +163,10 @@ var Attribute = Inheritable.inherit(/** @lends Attribute.prototype */{
         }
     },
 
+    destruct: function () {
```

Comment:
destruct function provides interface and default implementation for all attribute implementation, that actually does nothing ( https://github.com/dfilatov/inherit/blob/master/lib/inherit.js#L39 ).
It would work even without __base call,  but __base method call makes code to be written in generic way, that avoids  usage of additional information about Attribute parent class (that Attribute class's parent (Inheritable) has no destruct method)